packages/app/src/providers/hsds/HsdsProvider.tsx

```diff
@@ -2,31 +2,30 @@ import { type PropsWithChildren, useMemo } from 'react';
 
 import { type DataProviderApi } from '../api';
 import DataProvider from '../DataProvider';
+import { type Fetcher } from '../models';
 import { HsdsApi } from './hsds-api';
 
 interface Props {
   url: string;
-  username: string;
-  password: string;
   filepath: string;
   resetKeys?: unknown[];
+  fetcher: Fetcher;
```

Comment:
Note that the fetcher is required. HSDS requires authentication, so we can't initialise a basic fetcher like in `H5GroveProvider`.